pkg/mbedtls/Makefile.dep

```diff
@@ -0,0 +1,24 @@
+ifneq (,$(filter mbedtls_entropy,$(USEMODULE)))
+  USEMODULE += mbedtls_contrib
+  USEMODULE += hashes
+
+  # include sources if hardware is available
+  FEATURES_OPTIONAL += periph_hwrng
+  ifneq (,$(filter periph_hwrng,$(FEATURES_USED)))
+	  DEFAULT_MODULE += mbedtls_entropy_source_hwrng
+  endif
+
+  FEATURES_OPTIONAL += periph_adc
+  ifneq (,$(filter periph_adc,$(FEATURES_USED)))
+	  DEFAULT_MODULE += mbedtls_entropy_source_adc
+  endif
```

Comment:
What happens if the hardware provides both ? What is the entropy source selected in this case ?

Reply:
I think those are not mutually exclusive, @PeterKietzmann ?

Reply:
Then both are used (if not manually disabled). That is actually the intent of the entropy module, to accumulate multiple entropy sources.